### DISCUSSION TITLE
It may be the right time to recommend using `@use` instead of `@import`

### DISCUSSION THREAD

**Discussion by ericmorand** [twbs/bootstrap#39584] [General]
The sass team has emitted the formal recommendation of not using `@import` anymore a long time ago but Bootstrap documentation still display examples that use this keyword instead of `@use`.

This leads to some confusion by the community since some of us recommend to the developers to use the `@use` keyword while the Bootstrap documentation recommends otherwise.

This stack overflow answer for example, written by myself, has a legit comment by someone asking which is right - Bootstrap documentation or my answer.

https://stackoverflow.com/a/77814573/354490

I think that updating the documentation in favour of `@use` will prepare the community for the eventual removal of `@import` and would additionally encourage everybody to embrace the very clean and powerful modular approach that SASS has been proposing for years and that is largely ignored as of today.

**Comment by mdo** [comment]
Getting there to support `@use` requires a pretty decent rewrite of our codebase, so it won't happen until v6.

**Reply by ericmorand** [reply]
That's a fair point, but my point was about documentation: consuming Bootstrap from the `@use` directive works perfectly today and I feel there is no reason to not let the documentation reflect this situation.
